source/documentation/index.md

```diff
@@ -133,17 +133,20 @@ Follow this end-to-end journey to setup your developer environment, request NOP
 Example of a POST request for a single EORI number:
 
 	```code
+ 	{
```

Comment:
Sorry, just realised the code I gave you I've added two extra unneeded  brackets here